### PR TITLE
Base Notification Citation View

### PR DESCRIPTION
**Pull request by blakegreen32**
Refactored class names for notification views, removed settings in notifications view, added more properties to notification citation views

This foam PR must be merged and pegged before CircleCI will pass: https://github.com/foam-framework/foam2/pull/3452

### CODE REVIEW

File: src/foam/nanos/notification/NotificationCitationView.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/nanos/notification/NotificationCitationView.js]: 2020

Comment (kgrgreer) [reply src/foam/nanos/notification/NotificationCitationView.js]: How is this different than the NotificationNotifcationView?

Comment (blakegreen32) [reply src/foam/nanos/notification/NotificationCitationView.js]: NotificationNotificationView simply extends the NotificationCitationView, I don't see the point of it and I don't think its used anywhere.

**Review by nanoscott — APPROVED** [review]

File: src/foam/nanos/notification/NotificationView.js [hunk 2]
```diff
@@ -8,43 +8,124 @@ foam.CLASS({
   package: 'foam.nanos.notification',
   name: 'NotificationView',
   extends: 'foam.u2.View',
+  documentation: 'Notification controller',
 
-  axioms: [
-    foam.pattern.Faceted.create()
```
Comment (jlhughes) [inline src/foam/nanos/notification/NotificationView.js:13]: @kgrgreer can you comment on this.
Believe was important so a Notification view could be replaced with a custom view.

**Comment by nanoscott** [comment]
@googlebot I consent.

File: src/foam/nanos/notification/NotificationView.js (hunk 2 above)
Comment (blakegreen32) [reply src/foam/nanos/notification/NotificationView.js:13]: NotificationView was renamed to NotificationCitationView

Comment (blakegreen32) [reply src/foam/nanos/notification/NotificationView.js:13]: see NotificationCitationView

File: src/foam/nanos/notification/NotificationView.js [hunk 3]
```diff
@@ -8,43 +8,124 @@ foam.CLASS({
...
+  ],
+
+  requires: [
+    'foam.nanos.notification.Notification',
+    'foam.nanos.notification.NotificationRowView',
+    'foam.u2.stack.Stack'
+  ],
+
+  exports: [
+    'as data',
```
Comment (kgrgreer) [inline src/foam/nanos/notification/NotificationView.js]: Should extend Controller and then remove this line if this is a Controller not a View

**Comment by kgrgreer** [comment]
Has a conflict.

**Pull request merged**